packages/core/data-transfer/lib/providers/local-strapi-source-provider/__tests__/index.test.ts

```diff
@@ -120,4 +120,86 @@ describe('Local Strapi Source Provider', () => {
       });
     });
   });
+
+  describe('Streaming Schemas', () => {
+    test('Should successfully create a readable stream with all Schemas', async () => {
+      const contentTypes = {
+        foo: { uid: 'foo', attributes: { title: { type: 'string' } } },
+        bar: { uid: 'bar', attributes: { age: { type: 'number' } } },
+      };
+
+      const components ={
+        'basic.simple': {
+          collectionName: 'components_basic_simples',
+          info: { displayName: 'simple', icon: 'ambulance', description: '' },
+          options: {},
+          attributes: { name: {type:'string'} },
+          uid: 'basic.simple',
+          category: 'basic',
+          modelType: 'component',
+          modelName: 'simple',
+          globalId: 'ComponentBasicSimple'
+        },
+        'blog.test-como': {
+          collectionName: 'components_blog_test_comos',
+          info: {
+            displayName: 'test comp',
+            icon: 'air-freshener',
+            description: ''
+          },
+          options: {},
+          attributes: { name: { type: 'string' } },
+          uid: 'blog.test-como',
+          category: 'blog',
+          modelType: 'component',
+          modelName: 'test-como',
+          globalId: 'ComponentBlogTestComo'
+        },
+      }
+
+      const provider = createLocalStrapiSourceProvider({ getStrapi: getStrapiFactory({
+        contentTypes,
+        components
+      }) });
+
+      await provider.bootstrap();
+
+      const schemasStream = provider.streamSchemas() as Readable;
+      const schemas = await collect(schemasStream);
+
+      expect(schemasStream).toBeInstanceOf(Readable);
+      expect(schemas).toHaveLength(4);
+
+      expect(schemas).toEqual([
```

Comment:
Is it safe to assume it will always follow the same ordering?
Should this test break if for some reason we're using `[...Object.values(components), ...Object.values(contentTypes)]`?

Honestly, just open questions here